### ISSUE TITLE
erc20 example test failure

### ISSUE THREAD

**Issue by shunjizhan** [AcalaNetwork/bodhi.js#148]
```
 15 passing (46s)
  2 failing

  1) ACAToken
       Transfer adds amount to destination account:
     AssertionError: Expected "751907535" to be equal 7
      at Context.<anonymous> (test/ACAToken.test.ts:50:82)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)

  2) BasicToken
       Can not transfer from empty account:
     AssertionError: Expected transaction to be reverted, but other exception was thrown: Error: -32603: execution fatal: Module { index: 180, error: 2, message: Some("NoPermission") }
```

**Comment by shunjizhan** [comment]
more logs
<img width="1507" alt="Screen Shot 2021-11-26 at 10 17 13" src="https://user-images.githubusercontent.com/16548786/143516879-2717167a-0de4-48cd-aee9-01d1f4fc5007.png">

**Comment by shunjizhan** [comment]
fixed by #151
